DataFormats/METReco/src/classes_def.xml

```diff
@@ -164,8 +164,8 @@
   </class>
   <class name="edm::Wrapper<reco::EcalHaloData>"/>
 
-  <class name="reco::HcalHaloData" ClassVersion="10">
-   <version ClassVersion="10" checksum="3928363167"/>
+  <class name="reco::HcalHaloData" ClassVersion="11">
+   <version ClassVersion="11" checksum="624180652"/>
```

Comment:
old "ClassVersion-checksum" pairs should always be kept if there was data saved with them.
add ```<version ClassVersion="10" checksum="3928363167"/>``` right below what you have in L168